README.md

```diff
@@ -1,25 +1,22 @@
-
-We are collecting feedback on whether developers need public extensibility APIs for UpgradeAssistant to be able to write their own transformations. Please let us know in an issue in this repository or via the Visual Studio feedback channel.
+This repository is now home to 3rd-party extensions to the [Visual Studio Upgrade Assistant](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.upgradeassistant)
+and its [CLI](https://www.nuget.org/packages/upgrade-assistant#versions-body-tab) alternative.
 
 ## Status
 
 | |Build (Debug)|Build (Release)|
 |---|:--:|:--:|
 | ci |[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/upgrade-assistant/dotnet.upgrade-assistant?branchName=main&jobName=Windows_NT&configuration=Windows_NT%20Debug)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=953&branchName=main)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/upgrade-assistant/dotnet.upgrade-assistant?branchName=main&jobName=Windows_NT&configuration=Windows_NT%20Release)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=953&branchName=main)|
 | official | [![Build Status](https://dev.azure.com/dnceng/internal/_apis/build/status/dotnet/upgrade-assistant/dotnet-upgrade-assistant?branchName=main&stageName=Build&jobName=Windows_NT&configuration=Windows_NT%20Debug)](https://dev.azure.com/dnceng/internal/_build/latest?definitionId=949&branchName=main)|[![Build Status](https://dev.azure.com/dnceng/internal/_apis/build/status/dotnet/upgrade-assistant/dotnet-upgrade-assistant?branchName=main&stageName=Build&jobName=Windows_NT&configuration=Windows_NT%20Release)](https://dev.azure.com/dnceng/internal/_build/latest?definitionId=949&branchName=main)|
-<br/>
 
 ## Overview
 
-This project and tool enables automation of common tasks related to upgrading .NET Framework projects to the latest versions of .NET (STS, LTS, or Preview). See https://dotnet.microsoft.com/platform/support/policy/dotnet-core for more details on the specific versions. Note that this tool will not handle every aspect of upgrading your project(s). Manual work *will* be required after using the tool to complete the upgrade.
-
-The tool has 2 entry points: [Analyze](#analyze-solution-prior-to-upgrade) and [Upgrade](#upgrade-solution) to assist in understanding dependencies before upgrading and then with the actual changes to your project files, code files, and dependencies. There are also several ways to add more support to the tool by adding [extensions](#solution-wide-extension-management), [experimental features](#experimental-features), and [optional features](#optional-features).
+This project aims to bring extensibility to the dotnet Upgrade Assistant tool via [Package Maps](docs/Extensibility.PackageMaps.md) and [API Maps](docs/Extensibility.APIMaps.md), allowing
```

Comment:
>This project  [](http://example.com/codeflow?start=0&length=13)

maps is not the only extesnibility but one of them, , we just starting with it. Lets rephrase  it to some thing like "one of the extensibility points"